export default function ErrorTest(){
  throw new Error('This page crashes on purpose to test Error Boundary!')
}
